protocol-tests/src/main/java/fixtures/dpgcustomization/implementation/DpgClientImpl.java

```diff
@@ -677,7 +677,8 @@ public PollerFlux<BinaryData, BinaryData> beginLroAsync(String mode, RequestOpti
                                 .setContext(
                                         requestOptions != null && requestOptions.getContext() != null
                                                 ? requestOptions.getContext()
-                                                : Context.NONE)),
+                                                : Context.NONE)
+                                .setServiceVersion(null)),
```

Comment:
Can we skip setting this if the value is `null`? This looks a bit odd in generated code as this is a no-op.

Reply:
Hi Srikanta, thanks for your suggestion. I have the same concern with you previously. 
But I finally set to `null` because our current logic is to `setServiceVersion(this.getServiceVersion().getVersion())` when two conditions meet: 1. it's DPG 2. apiversion is in query param. And if we don't want to pass it, we need to update the [polling strategy template](https://github.com/Azure/autorest.java/blob/main/extension-base/src/main/java/com/azure/autorest/extension/base/plugin/JavaSettings.java#L948-L972) in `JavaSettings`, but in JavaSettings, we don't know if there is `api-version` in query param, So if it is DPG and `api-version` is not in query param, it still will be set to `null`. To make the logic for two conditions in one place, I add `setServiceVersion` to the default template and determine its value in `ClientMethodTemplate`.